### DISCUSSION TITLE
升级后docker无法启动/重启报错，配置的网站都没法启动查看。

### DISCUSSION THREAD

**Discussion by wosqqachen** [1Panel-dev/1Panel#7546] [Q&A]
升级v1.10.21-lts 之后docker出现问题，不能启动、之前一直好用的，现在新增文档里面的加速镜像也不能启动
![image](https://github.com/user-attachments/assets/9b9e1fb1-3a1f-43ec-95f1-dc34403aa9cf)

**Comment by wanghe-fit2cloud** [comment]
建议根据提示信息中的 `journalctl` 命令检查是否存在 `error` 相关的日志记录。
> 升级 1Panel 版本不会影响服务器上已安装的 Docker。
